config/locales/en.yml

```diff
@@ -785,14 +786,17 @@ en:
       pending_review: Pending review
       preview_card_providers:
         allowed: Links from this publisher can trend
+        description_html: These are domains, links from which are often shared on your server. Links will not trend publicly unless the domain of the link is approved. Your approval (or rejection) extends to subdomains.
```

Comment:
The first sentence is awkward and difficult to understand. Maybe “These are domains from which links are often shared on your server.”?